bin/citrea/tests/evm/tracing.rs

```diff
@@ -112,14 +112,14 @@ async fn tracing_tests() -> Result<(), Box<dyn std::error::Error>> {
     // It was replaced with the gas limit in our trace.
     let reth_json = serde_json::from_value::<CallFrame>(json![{
         "from": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
-        "gas": "0xbbdf",
+        "gas": "0xc477",
         "gasUsed": "0xba65",
         "to": "0xe7f1725e7734ce288f8367e1bb143e90bb3f0512",
         "input": "0xb7d5b6580000000000000000000000005fbdb2315678afecb367f032d93f642f64180aa30000000000000000000000000000000000000000000000000000000000000003",
         "calls": [
             {
                 "from": "0xe7f1725e7734ce288f8367e1bb143e90bb3f0512",
```

Comment:
debug trace gas changed?

weird

Reply:
Because it does `deploy_contract` which calls `eth_estimate_gas` inside.